lib/introspection.js

```diff
@@ -79,17 +79,21 @@ const parseComments = lines => {
       let [type, comment] = common.section(rest, ', ');
       const types = type.split(' | ').map(s => s.slice(1, s.length - 1));
       if (types.length === 1) {
-        if (!ALL_TYPES.find(s => types[0].startsWith(s))) {
+        type = types[0];
+        console.log(types[0]);
+        if (!(ALL_TYPES.has(type) ||
+            (ALL_TYPES.has(type.slice(0, type.length - 2)) &&
```

Comment:
You can omit these parens I think.

Reply:
The closing parenthesis was in the wrong place. Thanks, though.

Reply:
I mean. this can be:
```js
const arrType = type.slice(0, type.length - 2)
if (!ALL_TYPES.has(type) &&
   (!ALL_TYPES.has(arrType) || type.slice(type.length - 2) !== '[]')) {
```